chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -154,10 +154,9 @@ def softmax_cross_entropy(
             number of dimensions is greater than 2.
         t (Variable): Variable holding an int32 vector of ground truth labels.
             If ``t[i] == -1``, corresponding ``x[i]`` is ignored.
-        normalize (Variable): Variable holding a boolean value which
-            determines the normalization constant. If true, this function
-            normalizes the cross entropy loss across all instances. If else,
-            it only normalizes along a batch size.
+        normalize (bool): If true, this function normalizes the cross entropy
```

Comment:
Actually, isn't it that if ``True``, it normalizes w.r.t. non-ignored labels in the batch, and otherwise, w.r.t. all elements in the batch? https://github.com/pfnet/chainer/blob/3ca579ee17cb2ef328f3a8a33473d3c9b690db41/chainer/functions/loss/softmax_cross_entropy.py#L60-L63